stackslib/src/chainstate/coordinator/mod.rs

```diff
@@ -292,6 +292,7 @@ impl RewardSetProvider for OnChainRewardSetProvider {
                     return Ok(RewardSet::empty());
                 }
             }
+            StacksEpochId::Epoch30 => todo!(), // Need input here
```

Comment:
This should have the same behavior as Epoch24

Reply:
Okay, added 3.0 to the same match arm as 2.4